app/views/alchemy/admin/elements/_element.html.erb

```diff
@@ -25,7 +25,24 @@
         <div id="element_<%= element.id %>_errors" class="element_errors"></div>
 
         <div id="element_<%= element.id %>_content" class="element-content-editors">
-          <%= render element.contents %>
+          <% element.contents.select { |c| c.definition[:group].blank? }.each do |content| %>
```

Comment:
I think `group_by` would be very helpful here to still make use of the Rails collection partial renderer. 

Also the group header should be translated. Preferably on the ElementEditor level and could be used as grouping source.